src/signals/incident-management/containers/IncidentDetail/components/StatusForm/components/EmailPreview/EmailPreview.tsx

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022 Gemeente Amsterdam
+import type { FC } from 'react'
+import styled from 'styled-components'
+import { themeSpacing } from '@amsterdam/asc-ui'
+import FormFooter from 'components/FormFooter'
+import ModalHeader from '../ModalHeader/ModalHeader'
+
+const ModalContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+`
+
+const StyledFormFooter = styled(FormFooter)`
+  height: auto;
+  div {
+    padding-left: ${themeSpacing(4)};
+    div {
+      padding-left: 0;
+    }
+  }
+`
+
+const StyledIframe = styled.iframe`
+  border: none;
+  padding-left: ${themeSpacing(2)};
+  padding-bottom: ${themeSpacing(16)};
```

Comment:
These two lines can be combined as 
```
padding: ${themeSpacing(2, 0, 16)};
```

Where does the value 16 come from? Seems like an arbitrary, magic, number.

Reply:
its the formfooter height. I've made that clear now